extractor/filesystem/secrets/mariadb/credential.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mariadb
+
+import "strings"
+
+// Credentials contains mariadb credentials
+//
+// TODO: add location in some way
```

Comment:
There's an unresolved TODO here - Did you manage to fix this? Is the Section part for location indication?